cdk/visit/migration_scripts/migrate_data_to_2_tables.py

```diff
@@ -0,0 +1,61 @@
+"""
+This script will migrate data from the original table to the two
+new tables.
+"""
+
+import boto3
+import os
+
+# TODO: Update these to point to correct env variables.
+original_table_name = os.environ["TABLE_NAME"]
+visits_table_name = os.environ["VISITS_TABLE_NAME"]
+users_table_name = os.environ["USERS_TABLE_NAME"]
+
+
+def get_table(table_name: str) -> boto3.resources.base.ServiceResource:
+    """
+    Get a table resource from the boto3 resource manager.
+
+    Args:
+        table_name: The name of the table to get.
+
+    Returns:
+        A table resource.
+    """
+    dynamodb = boto3.resource('dynamodb', region_name='us-east-1')
+    table = dynamodb.Table(table_name)
+    return table
+
+
+def get_all_data(table: boto3.resources.base.ServiceResource) -> list:
+    """
+    Get all data from a table.
+
+    Args:
+        table: The table to get data from.
+
+    Returns:
+        A list of all data from the table.
+    """
+    response = table.scan()
```

Comment:
How large is our table? AFAIK this scan call has a max return payload of 1mb at which point you will need to handle pagination

Reply:
The table is 83.6 kilobytes so we shouldn't need to handle pagination.